### PR TITLE
Zinkal changes

### PR DESCRIPTION
**Pull request by zdthakker-dev**
user-history bug,logged-in client page and dashboard icon.

### CODE REVIEW

File: src/Pages/Account/LoggedInUsers/LoggedInUsers.jsx [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+          year: "numeric",
+          month: "long",
+          day: "numeric",
+        });
+        return <div title={formattedDate}>{formattedDate}</div>;
+      },
+      sortable: true,
+      sortField: "createdAt",
+    },
+  ].filter(Boolean);
```
Comment (ca-pikachu) [inline src/Pages/Account/LoggedInUsers/LoggedInUsers.jsx:187]: Why are we filtering this?

File: src/Pages/Account/LoggedInUsers/LoggedInUsers.jsx [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+      "users.csv"
+    );
+  };
+
+  const handleDelete = (id) => {
+    showAlert(id, removeRow);
+  };
+
+  useEffect(() => {
+    setData([]);
```
Comment (ca-pikachu) [inline src/Pages/Account/LoggedInUsers/LoggedInUsers.jsx:298]: `setData([])` is already called in the cleanup function so we can remove it from here.

File: src/Pages/Account/LoggedInUsers/LoggedInUsers.jsx [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+    if (searchQuery !== "") {
```
Comment (ca-pikachu) [inline src/Pages/Account/LoggedInUsers/LoggedInUsers.jsx:299]: I've added a comment for this change in the `UserActivity.jsx` 
We can go with the same flow here.

**Review by ca-pikachu — CHANGES_REQUESTED** [review]
@zdthakker-dev Please resolve the conflicts and the requested changes

File: src/Pages/Dashboard/Dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -297,6 +299,38 @@ export default function Dashboard() {
                 </Card.Body>
               </Card>
             </Col>
+            <Col lg={6} md={12} sm={12} xl={3}>
+
+
+              <Card className=" overflow-hidden">
+                {/* <Link to="/logged-in-users"> */}
```
[the remark this answers is not in the document]
Comment (zdthakker-dev) [reply src/Pages/Dashboard/Dashboard/Dashboard.jsx:306]: yes

**Review by ca-pikachu — APPROVED** [review]

**Pull request merged**